config/prow/config.yaml

```diff
@@ -556,6 +556,9 @@ branch-protection:
               protect: true
             kubetest2-aks:
               protect: true
+        cve-feed-osv:
+          exclude:
+            - "^vulns-" # don't protect branches created by CVE feed github action
```

Comment:
somehow this is a funny statement, but it matches the pattern we have so far :-)